src/field_theory/intermediate_field.lean

```diff
@@ -376,44 +375,46 @@ variables {S}
 section tower
 
 /-- Lift an intermediate_field of an intermediate_field -/
-def lift1 {F : intermediate_field K L} (E : intermediate_field K F) : intermediate_field K L :=
-  map E (val F)
+def lift {F : intermediate_field K L} (E : intermediate_field K F) : intermediate_field K L :=
+map E (val F)
 
-/-- Lift an intermediate_field of an intermediate_field -/
-def lift2 {F : intermediate_field K L} (E : intermediate_field F L) : intermediate_field K L :=
+instance has_lift {F : intermediate_field K L} :
+  has_lift_t (intermediate_field K F) (intermediate_field K L) := ⟨lift⟩
+
+section restrict_scalars
+variables (K) [algebra L' L] [is_scalar_tower K L' L]
+
+/-- Given a scalar tower `K`, `L'`, `L` of fields, reinterpret an `L'`-intermediate-field of `L` an
+as a `K`-intermediate-field. -/
```

Comment:
The notation with `/` is closer to typical maths, is that worth the slight confusion of a non-type `E` among the types `L, L', K`?
```suggestion
/-- Given a tower `L / E / L' / K` of field extensions, where `E : intermediate_field L' L`,
reinterpret `E` as a `K`-intermediate-field. -/
```

Reply:
I copied this docstring from the one for `subalgebra`; do you want me to change both?

Reply:
How do the new docstrings look?

Reply:
Better, thanks!